xyz-models/src/main/java/com/here/xyz/responses/changesets/ChangeNotification.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2017-2022 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.xyz.responses.changesets;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.here.xyz.models.geojson.implementation.FeatureCollection;
+import com.here.xyz.responses.XyzResponse;
+
+/**
+ * A Changeset represents a set of Feature modifications having been performed as one transaction, single authored,
+ * which can contain multiple operations like insertions, deletions and updates.
+ */
+@JsonInclude(Include.NON_DEFAULT)
+public class ChangeNotification extends XyzResponse<ChangeNotification> {
```

Comment:
Better extend Payload here, as this model will not be used as a service response.